### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by farzadmf** [hashicorp/vscode-terraform#255]
Hello,

I just installed this extension to test and I'm not sure if I'm missing something, but I tried language server enabled and disabled, but I still have issues:

I have create two simple files in a directory:
```terraform
# resource-group.tf
resource "azurerm_resource_group" "my-rg" {
  location = "East US"
  name     = "myresourcegroup"
}
```
```terraform
# network-interface.tf
resource "azurerm_network_interface" "mediagraph-nic" {
  location             = azurerm_resource_group.my-rg.location
  name                 = "my-nic"
  resource_group_name  = azurerm_resource_group.my-rg.name
}
```

But, in my `network-interface.tf`, I see errors:
![image](https://user-images.githubusercontent.com/5768792/69966488-8124e800-14e4-11ea-87d6-c14a581409d7.png)

It may be also worth noting that `terraform plan` is fine with `location` and `resource_group_name` variables set in `network-interface.tf`; it's only the extension that's complaining about them.

Also, when I run `terraform plan`, it tells me that `"ip_configuration"` is a required field that needs to be set, and I was expecting the extension to warn me about that.

I didn't manually change any settings, and I used `Terraform: Enable/Disable Language Server`, but in case it helps, I see these settings:
```jsonc
  "terraform.indexing": {
    "enabled": false,
    "liveIndexing": false,
    "delay": 500,
    "exclude": [
      ".terraform/**/*",
      "**/.terraform/**/*"
    ]
  },
  "terraform.languageServer": {
    "enabled": true,
    "args": []
  },
```

**Comment by saisurya237** [comment]
Could you kindly explain how to setup the language server for the extension and enable and how to check the configuration?. Thank you

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
